plugins/build.gradle.kts

```diff
@@ -4,9 +4,30 @@ plugins {
     id("com.gradle.plugin-publish") version "0.12.0" apply false
 }
 
+val properties = java.util.Properties()
+val file = project.rootProject.file("../gradle.properties")
+try {
+    file.inputStream().use { properties.load(it) }
+} catch (e: Throwable) {
+    throw Exception(
+        "Cannot read gradle.properties file from root project. " +
```

Comment:
Error message is too verbose. Could be one sentence.

`Can't read ${file.absolutePath}`